CONTRIBUTING.md

```diff
@@ -80,16 +83,14 @@ for common troubleshooting steps.
 
 There are three main pieces of Mina documentation:
 
-1. The [`docs`](docs/) directory, which has prose documentation of various sorts.
-2. The `README.md` files in various directories. These explain the contents of that
+1. The [https://github.com/o1-labs/docs2)](https://github.com/o1-labs/docs2) repository for the [Mina Protocol](https://docs.minaprotocol.com/) docs website.
+2. The `README.md` files in various directories explain the contents of that
    directory at a high level: the purpose of the library, design constraints, anything else
    specific to that directory.
-3. Inline code comments. There are unfortunately very few of these currently,
-   but this will slowly change. We are now running `ocamldoc` and the generated
-   documentation is browsable
-   [online](https://mina-docs.storage.googleapis.com/index.html).
+3. Inline code comments. We are working to provide more comments. We run `ocamldoc` and the generated 
+   [OCaml package documentation](https://mina-docs.storage.googleapis.com/index.html) is browsable.
 
-Changes to the software should come with changes to the documentation.
+Changes to the software require changes to the documentation. The Mina Docs use a docs-as-code workflow to develop and publish product documentation using the same tools and processes as software code development. All user-impacting updates on the codebase require corresponding updates to the relevant documentation in the docs repo. See the [Docs Contributing Guidelines](https://github.com/o1-labs/docs2/blob/main/CONTRIBUTING.md). 
```

Comment:
We're not here (yet!)  for required doc updates as part of the feature updates, but let's act like it and move in that direction, wdyt?